src/test/scala/cc/factorie/app/nlp/ner/TestNerTag.scala

```diff
@@ -0,0 +1,24 @@
+package cc.factorie.app.nlp.ner
```

Comment:
Thanks for the test demonstrating the bug, but I don't think we actually need it in Factorie since, as it stands, it is basically testing ++= on a class that inherits from ArrayBuffer